README.md

```diff
@@ -884,6 +885,7 @@ Table of Contents
   * [Fontshare](https://www.fontshare.com/) - is a free fonts service. It’s a growing collection of professional grade fonts that are 100% free for personal and commercial use.
   * [Befonts](https://befonts.com/) - Provides several unqiue fonts for personal or commercial use.
   * [Font of web](https://fontofweb.com/) - Identify all the fonts used on a website and how they are used.
+  * [Bunny Fonts](https://fonts.bunny.net/) - All the Google Fonts with Google Fonts drop-in compatible API. Privacy oriented!.
```

Comment:
Please list bunny.net services under one listing(for example, see cloudflare's listing).

Reply:
In which category should I put it?, I ask because the main product of bunny.net is CDN but CDN does not have any free tier.

Reply:
Just pick whichever you feel is the most important/most prominent service

Reply:
Done